src/test/java/teammates/test/driver/TestUrl.java

```diff
@@ -0,0 +1,25 @@
+package teammates.test.driver;
+
+import java.net.MalformedURLException;
+import java.net.URL;
+import teammates.common.util.Url;
+
+/**
+ * Utility methods for Url class used only in tests.
```

Comment:
On second thoughts, `Holds additional methods for {@link Url} used only in tests.`